src/main/java/com/undertheriver/sgsg/memo/domain/Memo.java

```diff
@@ -0,0 +1,36 @@
+package com.undertheriver.sgsg.memo.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import com.undertheriver.sgsg.common.domain.BaseEntity;
+import com.undertheriver.sgsg.foler.domain.Folder;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Memo extends BaseEntity {
+
+	@Id
+	@GeneratedValue
+	private long id;
+
+	private String content;
```

Comment:
```suggestion
        @Lob
	private String content;
```
대용량 텍스트엔 lob을 넣어주는게 좋겠군요

Reply:
`@Lob` 공부하겠습니당

Reply:
저도 잘은 모르는데,,ㅎㅎ 대용량 텍스트는 Lob으로 관리하는 것 같네요